account/account_test.go

```diff
@@ -533,7 +533,7 @@ func TestAddDeployAccountDevnet(t *testing.T) {
 	_, err = devnet.Mint(precomputedAddress, new(big.Int).SetUint64(10000000000000000000))
 	require.NoError(t, err)
 
-	resp, err := acnt.AddDeployAccountTransaction(context.Background(), tx)
+	resp, err := acnt.AddDeployAccountTransaction(context.Background(), rpc.BroadcastDeployAccountTxn{DeployAccountTxn: tx})
```

Comment:
Wouldn't the other tests for invoke and declare transaction need updating? If needed this will be addressed on https://github.com/NethermindEth/starknet.go/issues/452